src/Common/Commands.Common.Authentication.Abstractions/Interfaces/IAzureEnvironment.cs

```diff
@@ -112,6 +112,11 @@ public interface IAzureEnvironment : IExtensibleModel
         /// </summary>
         string BatchEndpointResourceId { get; set; }
 
+        /// <summary>
```

Comment:
As discussed, we should not make this change, instead we should add this into the ExtendedProperties for AzureEnvironemnt, as we did for DataLake OAuth support

Reply:
fixed